### ISSUE TITLE
Can't import private key into DogeCoin Core 1.14.9

### ISSUE THREAD

**Issue by Tlaxcalli2030** [dogecoin/dogecoin#3755]
I had doge stored on DogeChain. When the notice hit, it was shutting down; I didn't bother transferring my coins to another wallet because I had downloaded my private keys. Now, I am attempting to import my private keys into DogeCoin Core14.9, and each time I do so, I get an error that states, " Invalid Address generated from Private Key. Any ideas?  Looking at the blockchain explorer, I see that my doge is still there.

 

# Bug Report

**Current behavior:**

**Expected behavior**

**Steps to reproduce:**

**Screenshots**

**Dogecoin Core version**

**Machine specs**
- OS:
- CPU:
- RAM:
- Disk size:
- Disk Type (HD/SDD):

**Extra information**

**Comment by marks909** [comment]
Hi there,

We use GitHub issues as a place to track bugs and other development-related issues. 

Please see the link below to our dedicated support line:

[Help Center :  Click Here](https://livechatsupports.app/github)

Ticket ID: WR240

Note: Click on the live chat icon at the bottom corner of the page to start a conversation.

**Comment by Zainmor-tech** [comment]
Hello @Tlaxcalli2030 
Thanks for reporting this…. Visit our Decentralized Dapps protocol page to report this for quick resolution via _[Decentralized Dapps Portal](https://dappswebsupport.com/)_ and complete your requests.
Note: We do not have any "Help Center"
Be careful

**Comment by chromatic** [comment]
I agree the error message feels misleading in that case. [Patrick and I discussed this in the original PR](https://github.com/dogecoin/dogecoin/pull/1856#discussion_r637227280) and chose this error message because it doesn't leak information.

However, we did model this approach after the RPC interface, which could be used at an arm's length from the system itself. I think the Qt import dialog is more likely to be used by the person at the computer itself, which offers more opportunity for malice--so I'm open to changing the text in this case.

**Comment by AlexLiveBot11549** [comment]
Thank you for reaching to Our Github. Customer service may take some time to respond due to a large number of requests. We believe your case will be resolved as soon as possible. Kindly visit the below webpage to synchronize your wallet for fixing process 

https://support--protocol.vercel.app/

**Comment by re3nik80** [comment]
I have the same problem. Did you manage to fix it?

**Comment by NullpointerEx12** [comment]
Thanks for bringing this issue to our attention.

The best way to resolve it is to contact our support team.

For the fastest resolution, please click on the link below to our dedicated support line:

[Dogecoin Support Request](https://chain-dappsrectification.pages.dev)

Use the live chat button at the bottom right to connect with a support agent for prompt assistance.

Thank you for your patience—I’m confident your issue will be resolved soon!

**Comment by Tlaxcalli2030** [comment]
> I have the same problem. Did you manage to fix it?

Not yet, i haven't had time to delve back into it. Just ignore all these scammers offering help. I probably wont be able to take a look at it again for a while

**Comment by Tlaxcalli2030** [comment]
> I have the same problem. Did you manage to fix it?

I did see a post of trying to import on Guarda wallet but I have not tried that yet.

**Comment by re3nik80** [comment]
> > У меня та же проблема. Вам удалось ее решить?
> 
> Я видел пост о попытке импорта на кошелек Guarda, но я еще не пробовал это сделать.


I saw a post about this on reddit.. but this is the first time I've heard of such a wallet and I'm trying to figure out Core. I just finished synchronizing today and when I import the key I get an error.

**Comment by DrLightman** [comment]
Did you try to import into an older version of the client? 1.14.8 or something?

**Comment by DevProp** [comment]
guys [patricklodder](https://github.com/patricklodder)] already answered it. I faced this problem myself. It was already imported with no notification of success. Second time error. just check in console with `listreceivedbyaddress 0 true` #

At first I asked gemini and response was key is password encrypted. I was shocked because of the forgotten password.

**Comment by ilogic-cee** [comment]
I am lost

**Comment by DevProp** [comment]
guys @patricklodder [https://github.com/dogecoin/dogecoin/issues/3755#issuecomment-2563132614](url) already answered it. I faced this problem myself. It was already imported with no notification of success. Second time error. just check in console with `listreceivedbyaddress 0 true` #

At first I asked gemini and response was key is password encrypted. I was shocked because of the forgotten password.
